notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>NIT: skeleton</p><p>NIT: belongs</p><p> </p><p>Maybe the text can only mention 1) dim, 2) belongs, 3) random_point: generates a random point that lies on the manifold, leave the others for later (since we have not introduced tangent space yet?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I think i am going to leave the others in for completeness (for now) and just add a note about "things making more sense after you learn about tangent spaces in future notebooks". because i don't want them to see the other ones in the geomstats code and be confused or feel like we left something out.</p>

Reply:
<p>i added in tangent space to this notebook, so now we can keep is_tangent without fear of confusion</p>